deploy/ansible/roles/filebeat/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+shipper_name: default
+env: test
+project_name: iroha-bench
```

Comment:
As we use filebeat for ELK stack, please tell people somehow (maybe in README.md) how to compose iroha/deploy/ansible/roles/filebeat/defaults/main.yml and what are the requirements for ELK stack (prerequisites for logs collection). Are there any settings for logs parsing, that will help others?

Reply:
@neewy this role will be removed in the next commit as it is required only for development and production purposes. It is not useful at all for our community